resources/lang/en/settings.php

```diff
@@ -45,8 +45,12 @@
         ],
         'poller' => [
             'distributed' => 'Distributed Poller',
+            'graphite' => 'Datastore: Graphite',
+            'influxdb' => 'Datastore: InfluxDB',
+            'opentsdb' => 'Datastore: OpenTSDB',
             'ping' => 'Ping',
```

Comment:
@murrant Reordering here handles the Problem
It's a list..

Reply:
Nope: https://github.com/librenms/librenms/blob/master/resources/js/components/LibrenmsSettings.vue#L166

Reply:
As @PipoCanaja I could put the datastore name first.

Reply:
oh ok, i thought i "fixed" some of this with reordering it in section
mmh ... are subsections possible? just thinking about it
datastore name first sounds also nice :-)

Reply:
Fixed via: #11280

Reply:
you're the man ;-)